frontend/www/js/omegaup/components/problem/Mine.vue

```diff
@@ -47,20 +47,27 @@
         <table class="table mb-0">
           <thead>
             <tr>
+              <th scope="col" class="text-center"></th>
+              <th scope="col" class="text-center">{{ T.wordsID }}</th>
               <th scope="col" class="text-center">{{ T.wordsTitle }}</th>
               <th scope="col" class="text-center">{{ T.wordsEdit }}</th>
               <th scope="col" class="text-center">{{ T.wordsStatistics }}</th>
             </tr>
           </thead>
           <tbody>
             <tr v-for="problem in problems">
-              <td class="d-flex align-items-center">
+              <td class="align-middle">
                 <input
                   type="checkbox"
                   v-model="selectedProblems"
                   v-bind:disabled="problem.visibility === -10"
                   v-bind:value="problem.alias"
                 />
+              </td>
+              <td class="text-center align-middle">
```

Comment:
nit:

```suggestion
              <td class="text-right align-middle">
```

esta es una columna numérica, entonces tiene más sentido alinearlo a la derecha.

Reply:
Listo